src/main/java/com/synopsys/arc/jenkinsci/plugins/customtools/versions/ToolVersion.java

```diff
@@ -32,8 +32,8 @@
 public class ToolVersion implements Serializable {
     private String variableName;
     private @CheckForNull String defaultVersion;
-    private String actualVersion;
-    private String versionSource;
+    private @CheckForNull String actualVersion;
```

Comment:
ok fair enough for matching the existing code style... but IMHO the existing style is not a good plan... but anyway... not my plugin... this is just a rant on my behalf